### ISSUE TITLE
won't let me push

### ISSUE THREAD

**Issue by MaggiK** [pearselab/python-introduction-MaggiK#1]
Hi Will,
I currently cannot access the python introduction repository by pushing or through my settings. I clicked the link, cloned everything and couldn't push anything into the repository without the error: 

"Maggis-MacBook-Pro:python-introduction-MaggiK maggi$ git push
remote: Permission to pearselab/python-introduction-MaggiK.git denied to MaggiK.
fatal: unable to access 'https://github.com/pearselab/python-introduction-MaggiK.git/': The requested URL returned error: 403"

I started over and reset the folder origin, tried fixing this error myself but unfortunately made my problem worse. Now I can't see the repository in my settings/repositories. I can only see it when I click the link on canvas. I still receive the error and am slightly nervous I will really screw stuff up. 

I hope you had a good weekend. 
Thanks,
Maggi

**Comment by willpearse** [comment]
I'll take a look at this tomorrow morning. Worst case, you'll just have to
re-clone from the class repository. As long as you've got your work still
on your computer, nothing can go wrong - don't worry.

**Comment by willpearse** [comment]
Have the steps I outlined in my email helped at all?

**Comment by MaggiK** [comment]
I tried a couple times. I also reentered my password and username but still
receive the error. I also tried via the git hub desktop but receive a
similar error.

Not sure what to try next.

Thanks,
Maggi

**Comment by willpearse** [comment]
Hang on - are you talking about the steps I outlined in my email? You're getting a username error when you ```git clone```?

**Comment by MaggiK** [comment]
Yep I am following your steps. I can clone the repository fine. It is only
when I push up to the repository.

**Comment by willpearse** [comment]
OK, could you pop over to https://appear.in/usubioprogramming for me? I'm going to have to delete this repository and get you to start over, and that'll be a lot easier if you're "on the line" with me
